generator/views/glean_ping_view.py

```diff
@@ -294,6 +295,24 @@ def _make_dimension(
             ],
         }
 
+        if lookml["type"] == "time":
+            # Remove any _{type} suffix from the dimension group name because each timeframe
+            # will add a _{type} suffix to its individual dimension name.
+            lookml["name"] = re.sub("_(date|time(stamp)?)$", "", looker_name)
+            lookml["timeframes"] = [
+                "raw",
+                "time",
+                "date",
+                "week",
+                "month",
+                "quarter",
+                "year",
+            ]
+            # Dimension groups should not be nested (see issue #82).
+            lookml["label"] = f"{group_label}: {group_item_label}"
+            del lookml["group_label"]
+            del lookml["group_item_label"]
```

Comment:
rather than adding and deleting, consider removing from the defn above and adding an else clause here

Reply:
I actually wrote it that way at first, but ended up deciding this approach was more straightforward/understandable (datetime metrics are only a minor edge case, and the explanatory comment I started writing for the else-group-label logic was annoyingly convoluted compared to the simple "Dimension groups should not be nested" explanation for this logic).